data/mil2525d/utilities/style-utilities/style-file-utilities/CreateAllMilitaryStyles.bat

```diff
@@ -1,57 +0,0 @@
-:: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
I guess this is being removed because there was also a copy in the csv2ArcGISStyle deployment zip - but that does raise the question - where/how does this file get in the deployment zip and which/where is the managed version of this file